src/content/docs/query-your-data/explore-query-data/dashboards/prebuilt-dashboards.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Pre-built dasboards
+tags:
+  - Query your data
+  - Explore and query data
+  - Dashboards
+  - Pre-built dashboards
+---
+
+import dashboardsDashboardIndex from 'images/dashboards_screenshot-full_dashboard-index.png'
+
+import dashboardsPrebuiltViewAll from 'images/dashboards_screenshot-full_prebuilt-view-all.png'
+
+import dashboardsPrebuiltSearch from 'images/dashboards_screenshot-full_prebuilt-search.png'
+
+
+Through the New Relic UI, you can access a collection of pre-built dashboards based on high levels of user interest and activity. These pre-built dashboards enhance quickstart dashboard discovery, speed up the installation process, and get to instantly visualize your data.
```

Comment:
Please consider rephrasing to:
`Through the New Relic UI, you can access a collection of commonly used pre-built dashboards, to help you instantly visualize your data.`

Reply:
Much better, thanks!